_posts/2023-09-18-the-definitive-guide-2-cn.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title:  "Chinese edition of Trino: The Definitive Guide"
+author: Manfred Moser, Martin Traverso, Matt Fuller
+image: /assets/ttdg2-cn-cover.png
+excerpt_separator: <!--more-->
+show_pagenav: false
+---
+
+Trino, Trino, Trino everywhere. Just looking at our website stats and the users
+in our community chat, we know that Trino is going places. We also know that one
+of these places with a large user community is China. And now we have good
+news for you.
+
+<!--more-->
+
+Today, we are happy to announce that a translation of the book [Trino: The
+Definitive Guide]({{ site.url }}/trino-the-definitive-guide.html) to Chinese is
+now available for the communities all across China and far beyond. We invite you
+all to get your own copy:
+
+<div class="card-deck spacer-30">
+    <a class="btn btn-pink" href="https://product.dangdang.com/11487789827.html">
+        Trino权威指南(原书第2版) 机械工业出版社
+    </a>
+</div>
+<div class="spacer-30"></div>
+
+Our thanks for making this happen go out the teams at O'Reilly and dangdang. We
+hope many readers will benefit from the translated edition.
+
+We look forward to chat to many of our new readers and Trino users on the
+`general-cn` channel on [community chat]({{site.baseurl}}/slack.html), other
+channels, and direct messaging.
+
+Also don't forget to tell us about your usage of Trino as guest on [Trino
```

Comment:
```suggestion
Also, don't forget to tell us about your usage of Trino as guest on the [Trino
```

Reply:
Should we say "reach out to us on Slack if you'd like to be a guest" or something to that effect? I think it's unclear to readers how they'd be a guest.

Reply:
Done